docs/docs/databases/sql-server.mdx

```diff
@@ -15,6 +15,15 @@ The connection string for SQL Server looks like this:
 mssql+pymssql://<Username>:<Password>@<Host>:<Port-default:1433>/<Database Name>/?Encrypt=yes
 ```
 
+You might need to install pymssql if you're running superset with docker-compose. 
+
+1. Enter docker container `docker exec -it superset_app bash`
+2. `pip install pymssql`
+3. exit container and run `docker-compose restart`
```

Comment:
This doesn't seem specific to mssql. There's already docs covering how to install packages in docker-compose https://github.com/apache/superset/tree/master/docker#local-overrides